packages/fxa-dev-launcher/profile.mjs

```diff
@@ -120,10 +121,8 @@ const fxaProfile = {
   'identity.fxaccounts.autoconfig.uri': fxaEnv.content,
   // disable password auto-fill; we typically don't need to test this daily.
   'signon.rememberSignons': false,
-  ...(process.env.FXA_DESKTOP_CONTEXT === 'oauth_webchannel_v1' && {
-    'identity.fxaccounts.oauth.enabled': true,
-    'identity.fxaccounts.contextParam': 'oauth_webchannel_v1',
-  }),
+  'identity.fxaccounts.oauth.enabled':
+    FXA_DESKTOP_CONTEXT === 'oauth_webchannel_v1',
```

Comment:
Not directly related to the issue, but I want this piece of logic since that conf value is true by default now in Firefox.

Reply:
That's set at https://github.com/mozilla/fxa/pull/19116/files/aa91704f7786402e42a6754656965ce116233e25#diff-4bd71824fbb7b7859dea0658fe0e26079ceac45877cbc224553d32e8735c77c8L108 which takes the value from lines 50-51.